### PR TITLE
Add S2S VPN via PowerShell doc

### PR DESCRIPTION
**Pull request by BaileyLawson**

### CODE REVIEW

**Comment by BaileyLawson** [comment]
I also need someone to add themselves as reviewer for this doc, @cblackuk maybe?

File: articles/azure/azs-how-create-S2S-VPN-ps.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+| \$AzureLocalGWName      | Name of local network gateway to create in public Azure                                    | <form oninput="result.value=AzureLocalGWName.value" id="AzureLocalGWName" style="display: inline;" ><input  type="text" id="AzureLocalGWName" name="AzureLocalGWName" style="display: inline;" placeholder="S2S-LNG"/></form> |
+| \$AzureGWConnectionName | Name of virtual network gateway connection to create in public Azure                       | <form oninput="result.value=AzureGWConnectionName.value" id="AzureGWConnectionName" style="display: inline;" ><input  type="text" id="AzureGWConnectionName" name="AzureGWConnectionName" style="display: inline;" placeholder="S2S-Connection"/></form> |
+| \$SharedKey             | Encryption key to use for VPN connection                                                   | <form oninput="result.value=SharedKey.value" id="SharedKey" style="display: inline;" ><input  type="text" id="SharedKey" name="SharedKey" style="display: inline;" placeholder="Password123!"/></form> |
+
+## Create site-to-site VPN connection from Azure Stack to Azure
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-ps.md]: Can we make this a bit clearer? Is it from UKCloud for Microsoft Azure to Azure public?

File: articles/azure/azs-how-create-S2S-VPN-ps.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+Connect-AzureRmAccount -EnvironmentName "AzureStackUser" -AccessToken $AzsAccessToken -AccountId $AzsContext.Account.Id
+
+## Set the public IP on the local network gateway
+$AzsLocalGW.GatewayIpAddress = $AzurePublicIP.IpAddress
+
+## Update the local network gateway
+Set-AzureRmLocalNetworkGateway -LocalNetworkGateway $AzsLocalGW
+</code></pre>
+
+After a short amount of time, the connection between Azure and Azure Stack should change to **Connected**.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-S2S-VPN-ps.md]: Similarly to the above, can we make sure it's clear which version of Azure/Azure Stack we're talking about

File: articles/azure/azs-how-create-S2S-VPN-ps.md (hunk 1 above)
Comment (BaileyLawson) [reply articles/azure/azs-how-create-S2S-VPN-ps.md]: Have changed "Azure" to "public Azure" which I think clears things up?

File: articles/azure/azs-how-create-S2S-VPN-ps.md (hunk 2 above)
Comment (BaileyLawson) [reply articles/azure/azs-how-create-S2S-VPN-ps.md]: Have changed "Azure" to "public Azure" which I think clears things up?

**Pull request merged**